mentor_classifier/mentor_classifier/api.py

```diff
@@ -184,6 +217,14 @@ def fetch_mentor_data(mentor: str) -> dict:
     return data
 
 
+def fetch_me_data(cookies: Dict[str, str] = {}, headers: Dict[str, str] = {}) -> dict:
```

Comment:
change this to `def fetch_mentor_answers_and_name() -> Tuple[List[AnswerInfo], str]` (see above)